script/check_version

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+echo
+echo "Built packages have the following versions:"
+echo $(cat CHANGELOG.md | sed -n 's/.*\(\([[:digit:]]\.\)\{2\}[[:digit:]]\).*/\1/p' | head -n 1
+) -- CHANGELOG.md
+echo $(monodis --assembly bin/Instrumental.dll | grep Version | sed -n 's/.*\(\([[:digit:]]\.\)\{3\}[[:digit:]]\).*/\1/p') -- Instrumental.dll '(from src/Properties/AssemblyInfo.cs)'
+echo $(ls bin | sed -n 's/Instrumental\.\(\([[:digit:]]\.\)\{2\}[[:digit:]]\)\.nupkg/\1/p' | tail -n 1) -- Nuget Package version '(Instrumental.nuspec)'
+echo "If these do not match, please correct the source and rebuild"
```

Comment:
It should, yes.  It should be easy to truncate that, I don't think we will ever include build number.